app/models/miq_alert_status_action.rb

```diff
@@ -15,8 +15,9 @@ class MiqAlertStatusAction < ApplicationRecord
   after_save :update_status_acknowledgement
 
   def only_assignee_can_acknowledge
-    if action_type == 'acknowledge' && miq_alert_status.assignee.try(:id) != user.id
-      errors.add(:user, "that is not assigned cannot acknowledge")
+    if ['acknowledge', 'unacknowledge', 'hide', 'show'].include?(action_type) &&
```

Comment:
There is a bug fix hiding here, only assignee should be able to 'unacknowledge' just like with 'acknowledge'
I will move this to a separate PR.